torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -199,6 +199,52 @@ def _annotate_conv_node_helper(
                 input_qspec_map=input_qspec_map, _annotated=True
             )
 
+    def _annotate_linear_node_helper(
+        self,
+        linear_node: torch.fx.Node,
+        annotate_output: bool,
+        quantization_config: QuantizationConfig,
+    ) -> None:
+        """Helper function to annotate the linear node"""
+        input_qspec_map = {}
+        assert linear_node.target in (
+            torch.ops.aten.mm.default,
+            torch.ops.aten.addmm.default,
+        )
+        has_bias = linear_node.target is torch.ops.aten.addmm.default
+        input_index = 1 if has_bias else 0
+        weight_index = input_index + 1
+
+        input_node = linear_node.args[input_index]
+        assert isinstance(input_node, Node)
+        input_qspec_map[input_node] = get_input_act_qspec(quantization_config)
+
+        t_node = linear_node.args[weight_index]
+        assert isinstance(t_node, Node)
```

Comment:
I think switching to use the new capture will be easier, this matching is not covering all use cases since there could be sym_size ops etc. I think ideally you can wait until https://github.com/pytorch/pytorch/pull/107872 and change the capture method

Reply:
Same here. Is it OK to land this first then we switch to new API later? Thanks.